packages/components/ng-ovh-payment-method/src/components/integration/component/adyen/service.js

```diff
@@ -0,0 +1,36 @@
+export default {
+  constructPostParams: (state) => {
+    // stringify the adyen component data
+    const resultStr = JSON.stringify({
+      ...{
+        ...state.data,
+        origin: window.location.origin,
+      },
+    });
+    // base64 the adyen component data
+    const formData = btoa(resultStr);
+    return {
+      formData,
+    };
+  },
+
+  getAdyenCallbackUrlParam: (callbackUrlParams) => {
+    return {
+      MD: callbackUrlParams.MD,
+      PaRes: callbackUrlParams.PaRes,
+      paymentMethodId: callbackUrlParams.paymentMethodId,
+      transactionId: callbackUrlParams.transactionId,
+    };
+  },
```

Comment:
I'm having issue understanding the purpose of this function, looks like a `_.pick` would do the job

Reply:
Method was previously used elsewhere. Will remove it